client/src/components/modal.jsx

```diff
@@ -1,32 +1,39 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import styles from '../styles/modal.css';
 
 
-const Modal = (data) => {
-  const clickedIdx = 0;
-  return (
-    <div className={styles.modal}>
-      <div className={styles.modalContent}>
-        <div className={styles.modalImage}>
-          <img src={data.dataArr[6].photosAndComments[clickedIdx].imageUrl} alt="host home" />
+const Modal = (props) => {
```

Comment:
We can destructure everything passed into props directly in the function argument 
`const Modal = ({ dataArr, toggleModal, displayModal, clickedIdx, handleModalNextButton, handleModalPreviousbutton }) => {` 

this will save you from having to write `props.` every time. 

> 

 

> 

>